calibration/FLORTO/CGINS-FLORTO-01342__20201015.csv

```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1342,CC_angular_resolution,1.076,
+1342,CC_depolarization_ratio,0.039,
+1342,CC_measurement_wavelength,700,
+1342,CC_scattering_angle,124,
```

Comment:
looks good

Reply:
These measure scattering at three wavelengths... not sure why we only have the 700 nm listed.